### ISSUE TITLE
YoloV7 custom net-scale-factor

### ISSUE THREAD

**Issue by wirsu** [marcoslucianops/DeepStream-Yolo#265]
Hi, 
Have a question on using custom `net-scale-factor` in deepstream config for `YOLOV7`

I understand the net-scale-factor of 0.00392... is used because these are derived from the default imagenet parameters and am aware it can be changed in the deepstream configs
I have successfully trained YoloV7 models and inference runs well but I want to use custom normalization params for images. 

How can I train yolov7 with custom normalization, and thus use a custom netscale factor 

Here in YOLOV7 repo I see the mentioning of such params but no clear way to change them.
https://github.com/WongKinYiu/yolov7/blob/9a00c173a44bfcf3a58cc2a60e37deb6a1fb9e03/utils/torch_utils.py#L232-L237

I'm also aware that because of good batch normalization, initial image normalization isn't a huge issue. 
But i'm conscientious that my dataset is very out of distribution compared to the imagenet dataset. 

As expected, just changing the net-scale-factor in the configs leads to bad results. 
Thanks

**Comment by marcoslucianops** [comment]
The `net-scale-factor=0.0039215697906911373` means no normalization (`1/255`).

To calculate the net-scale-factor value (example for std=[0.229, 0.224, 0.225])

```
import numpy as np
std = np.array([0.229, 0.224, 0.225]).mean() * 255
print(1/std)
```

You need to set the offsets value (example for mean=[0.485, 0.456, 0.406])
```
import numpy as np
mean = np.array([0.485, 0.456, 0.406]) * 255
print(";".join(str(x) for x in mean))
```

Example in `config_infer_primary.txt` file:

```
net-scale-factor=0.0173520735727919486
offsets=123.675;116.28;103.53
```
